### ISSUE TITLE
HTML not displayed on auth page in HF Spaces

### ISSUE THREAD

**Issue by ARusterholz-edu** [gradio-app/gradio#7913]
### Describe the bug

When using the auth page for login, it is possible to show HTML in an auth_message.

<img width="1204" alt="image" src="https://github.com/gradio-app/gradio/assets/147533556/52abda59-4d1b-4810-ba31-c50ac83790ca">

Shown above working in Colab Notebook. However, this feature seems to not be rendering HTML properly in huggingface space which is using the same code block:

<img width="571" alt="image" src="https://github.com/gradio-app/gradio/assets/147533556/3d51e08a-f69a-4b07-bca7-91622b607d96">

### Have you searched existing issues?  🔎

- [X] I have searched and found no existing issues

### Reproduction

```python
import gradio as gr

def greet(name):
    return f"Hello, {name}!"

auth_message = """
    <h2>Welcome to our Application!</h2>
    <p>Please log in to use this application:</p>
    <ul>
        <li>Log in using your <strong>username</strong> and <strong>password</strong>.</li>
        <li>If you don't have an account, please sign up.</li>
    </ul>
"""

def same_auth(username, password):
    return username == password
demo = gr.Interface(fn=greet, inputs="text", outputs="text")

demo.launch(auth=same_auth, auth_message=auth_message)


```

### System Info

```shell
Honestly not sure how to pull that information from a HuggingFace space
```

### Severity

I can work around it

**Comment by ARusterholz-edu** [comment]
RUN pip install --no-cache-dir 	gradio[oauth]==4.19.2 	"uvicorn>=0.14.0" 	spaces


Seems to be the relevant gradio info from the huggingface space

**Comment by ARusterholz-edu** [comment]
[#7545](https://github.com/gradio-app/gradio/pull/7545) [1fa2e91](https://github.com/gradio-app/gradio/commit/1fa2e914ca4663a47743cc34acdf98c7fcc469c8) - Fixes auth_message so that it correctly renders HTML. Thanks [@abidlabs](https://github.com/abidlabs)!


Ah, I see now. This has been fixed as of 4.20
